src/clients/c++/perf_analyzer/main.cc

```diff
@@ -276,8 +277,13 @@ Usage(char** argv, const std::string& msg = std::string())
   std::cerr
       << FormatMessage(
              " --service-kind: Describes the kind of service perf_analyzer "
-             "to generate load for. The options are \"triton\" and "
-             "\"tfserving\". Default value is \"triton\".",
+             "to generate load for. The options are \"triton\", \"tfserving\" "
+             "and \"torchserve\". Default value is \"triton\". Note in order "
+             "to use \"torchserve\" backend --input-data option must point to "
+             "a json file holding data in the following format {\"data\" : "
+             "[{\"TORCHSERVE_INPUT\" : [\"<complete path to the content "
+             "file>\"]}, {...}...]}. The type of file here will depend on the "
```

Comment:
Isn't the type of file .json for all models? Do you mean structure of the json?

Reply:
The torchserve is very restrictive and only allows a single input in the hosted model. The content for the tensor is provided through s file(usually a jpg or txt). I am having some issues in trying to directly send out the bytes from the file as the HTTP body. 
The difference here is, instead of pointing to the raw_content in the field for TORCHSERVE_INPUT, we will insert the complete path to the file to use for creating request inputs. This will probably change with the later PR through which we can include raw_contents of the file in json. Need to perform some performance analysis before making the switch though.